tck/optional/src/main/java/org/eclipse/microprofile/metrics/test/optional/MpMetricOptionalTest.java

```diff
@@ -1256,22 +1077,6 @@ public void testValidateGetJSONParam() throws InterruptedException {
                         + JSON_BASE_REQUEST_END, nullOrGreaterThanZero())
                 .body(JSON_BASE_MIN_TIME_START + "getMultipleParam2" + STRING_PARAM + LIST_PARAM
                         + JSON_BASE_REQUEST_END, nullOrGreaterThanZero())
-                .body(JSON_BASE_REQUEST_COUNT_START + "getMultipleParam3" + BOOLEAN_PARAM + BOOLEANW_PARAM
-                        + DOUBLE_PARAM + STRING_PARAM + ARRAY_BRACKETS + JSON_BASE_REQUEST_END, equalTo(1))
-                .body(JSON_BASE_REQUEST_TIME_START + "getMultipleParam3" + BOOLEAN_PARAM + BOOLEANW_PARAM + DOUBLE_PARAM
-                        + STRING_PARAM + ARRAY_BRACKETS + JSON_BASE_REQUEST_END, not(0))
-                .body(JSON_BASE_MAX_TIME_START + "getMultipleParam3" + BOOLEAN_PARAM + BOOLEANW_PARAM + DOUBLE_PARAM
-                        + STRING_PARAM + ARRAY_BRACKETS + JSON_BASE_REQUEST_END, nullOrGreaterThanZero())
-                .body(JSON_BASE_MIN_TIME_START + "getMultipleParam3" + BOOLEAN_PARAM + BOOLEANW_PARAM + DOUBLE_PARAM
-                        + STRING_PARAM + ARRAY_BRACKETS + JSON_BASE_REQUEST_END, nullOrGreaterThanZero())
-                .body(JSON_BASE_REQUEST_COUNT_START + "getMultipleParam4" + SET_PARAM + SORTED_SET_PARAM + DOUBLE_PARAM
```

Comment:
I would recommend removing the double array from this test, but leaving it in.  IIUC, this (and it's POST counterpart) are the only tests that verify `Set` and `SortedSet` parameters (which are legal in the JAX-RS 2.1 spec).  So either we should use these methods but remove the arrays or we should create new methods to test Set/SortedSet.

Reply:
should be done now, please check